hibernate-core/src/main/java/org/hibernate/type/CollectionType.java

```diff
@@ -194,21 +195,21 @@ public String toLoggableString(Object value, SessionFactoryImplementor factory)
 				}
 			}
 		}
```

Comment:
IIUC, by removing:

`if ( !Hibernate.isInitialized( value ) ) {
     return "<uninitialized>";
}`

I believe that `renderLoggableString` will initialize the collection if was uninitialized.